srcs/ServerCmds/privmsg.cpp

```diff
@@ -0,0 +1,55 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   privmsg.cpp                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mat <mat@student.42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/07 14:47:09 by mat               #+#    #+#             */
+/*   Updated: 2023/12/08 10:03:31 by mat              ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "Server.hpp"
+
+void Server::privmsg(
+	const std::vector<std::string> &cmd, Client *const client) {
+	const size_t size = cmd.size();
+	if (size < 3) {
+		if (size < 2)
+			Utils::sendFormattedMessage(ERR_NEEDMOREPARAMS, client);
+		else
+			Utils::sendFormattedMessage(ERR_NOTEXTTOSEND, client);
+		return;
+	}
+
+	const std::string fullMessage =
+		Utils::getFullMessage(cmd, PRIVMSG_START_INDEX);
+	std::vector<std::string> receiverList =
+		Utils::splitString(cmd[1], CMD_ARG_SEPARATOR);
+	for (std::vector<std::string>::iterator itV = receiverList.begin();
```

Comment:
`for (std::vector<std::string>::const_iterator itTarget = receiverList.begin();`